projects/canopy/src/lib/repsonsive/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './show-at/index';
```

Comment:
I'm wondering if these should be part of the grid module and same folder since they are strictly related.

Reply:
On one hand it does make sense to be part of the grid module, but then again - you might want to use these responsive directives without necessarily using the grid, in that sense maybe it's not a good to couple them together like that. Also, you may use the grid in your app, but never use these directives 🤔 

Alternatively, we could see them as another util directive and put them in the **/util-directives** folder. I was seeing them similar to Bootstrap's display utility classes. What do you think?

Would be great to get some more perspectives CC @andy-polhill  @paul-request  @aaronBery

Reply:
Yes, that's a good point. 
I think they should go into `/util-directives`.

Reply:
I can see them being quite multi purpose, personally I'm happy for them to sit in their own module.